src/bitcoin-tx.cpp

```diff
@@ -482,19 +482,30 @@ static void MutateTxDelOutput(CMutableTransaction& tx, const std::string& strOut
     tx.vout.erase(tx.vout.begin() + outIdx);
 }
 
-static const unsigned int N_SIGHASH_OPTS = 6;
+static const unsigned int N_SIGHASH_OPTS = 12;
 static const struct {
     const char *flagStr;
     int flags;
-} sighashOptions[N_SIGHASH_OPTS] = {
+} 
+sighashOptions[N_SIGHASH_OPTS] = {
     {"ALL", SIGHASH_ALL},
     {"NONE", SIGHASH_NONE},
     {"SINGLE", SIGHASH_SINGLE},
-    {"ALL|ANYONECANPAY", SIGHASH_ALL|SIGHASH_ANYONECANPAY},
-    {"NONE|ANYONECANPAY", SIGHASH_NONE|SIGHASH_ANYONECANPAY},
-    {"SINGLE|ANYONECANPAY", SIGHASH_SINGLE|SIGHASH_ANYONECANPAY},
+    {"ALL|ANYONECANPAY", SIGHASH_ALL | SIGHASH_ANYONECANPAY},
+    {"NONE|ANYONECANPAY", SIGHASH_NONE | SIGHASH_ANYONECANPAY},
+    {"SINGLE|ANYONECANPAY", SIGHASH_SINGLE | SIGHASH_ANYONECANPAY},
+    {"ALL|FORKID", SIGHASH_ALL | SIGHASH_FORKID},
+    {"NONE|FORKID", SIGHASH_NONE | SIGHASH_FORKID},
+    {"SINGLE|FORKID", SIGHASH_SINGLE | SIGHASH_FORKID},
+    {"ALL|FORKID|ANYONECANPAY",
```

Comment:
Imho it's better readable, if it's on one line. Same applies to `core_write.cpp`.

Reply:
agree